Test failure: `com.linecorp.armeria.internal.client.dns.DefaultDnsResolverTest.[2] resolvedAddressType=IPV6_PREFERRED`

```
java.lang.AssertionError: 
Expecting actual:
  4.99588634S
to be greater than or equal to:
  5S

	at com.linecorp.armeria.internal.client.dns.DefaultDnsResolverTest.resolveLessPreferredQuestionFirst(DefaultDnsResolverTest.java:185)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at java.base/java.util.stream.ForEachOps$ForEachOp$OfRef.accept(ForEachOps.java:183)
	at java.base/java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:197)
	at java.base/java.util.stream.ReferencePipeline$2$1.accept(ReferencePipeline.java:179)
	at java.base/java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:197)
	at java.base/java.util.stream.ForEachOps$ForEachOp$OfRef.accept(ForEachOps.java:183)
	at java.base/java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:197)
	at java.base/java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:197)
	at java.base/java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:197)
	at java.base/java.util.ArrayList$ArrayListSpliterator.forEachRemaining(ArrayList.java:1625)
	at java.base/java.util.stream.ReferencePipeline$Head.forEach(ReferencePipeline.java:762)
	at java.base/java.util.stream.ReferencePipeline$7$1.accept(ReferencePipeline.java:276)
	at java.base/java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:197)
	at java.base/java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:197)
	at java.base/java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:197)
	at java.base/java.util.ArrayList$ArrayListSpliterator.forEachRemaining(ArrayList.java:1625)
	at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:509)
	at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:499)
	at java.base/java.util.stream.ForEachOps$ForEachOp.evaluateSequential(ForEachOps.java:150)
	at java.base/java.util.stream.ForEachOps$ForEachOp$OfRef.evaluateSequential(ForEachOps.java:173)
	at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
	at java.base/java.util.stream.ReferencePipeline.forEach(ReferencePipeline.java:596)
	at java.base/java.util.stream.ReferencePipeline$7$1.accept(ReferencePipeline.java:276)
	at java.base/java.util.ArrayList$ArrayListSpliterator.forEachRemaining(ArrayList.java:1625)
	at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:509)
	at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:499)
	at java.base/java.util.stream.ForEachOps$ForEachOp.evaluateSequential(ForEachOps.java:150)
	at java.base/java.util.stream.ForEachOps$ForEachOp$OfRef.evaluateSequential(ForEachOps.java:173)
	at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
	at java.base/java.util.stream.ReferencePipeline.forEach(ReferencePipeline.java:596)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)

```

--- a/core/src/test/java/com/linecorp/armeria/internal/client/dns/DefaultDnsResolverTest.java
+++ b/core/src/test/java/com/linecorp/armeria/internal/client/dns/DefaultDnsResolverTest.java
@@ -155,8 +155,6 @@ class DefaultDnsResolverTest {
                             NoopDnsCache.INSTANCE, eventLoop, ImmutableList.of(), 1,
                             queryTimeoutMillis, HostsFileEntriesResolver.DEFAULT);
 
-            final DnsQuestionContext ctx = new DnsQuestionContext(eventLoop, queryTimeoutMillis);
-
             final Stopwatch stopwatch = Stopwatch.createStarted();
             final List<DefaultDnsQuestion> questions;
             if (resolvedAddressType == ResolvedAddressTypes.IPV4_PREFERRED) {
@@ -169,9 +167,9 @@ class DefaultDnsResolverTest {
                         new DefaultDnsQuestion("foo.com.", DnsRecordType.A));
             }
 
-            // resolver.resolveAll() should be executed by the event loop set to DnsNameResolver.
+            // resolver.resolve() should be executed by the event loop set to DnsNameResolver.
             final CompletableFuture<List<DnsRecord>> result = eventLoop.submit(() -> {
-                return resolver.resolveAll(ctx, questions, "");
+                return resolver.resolve(questions, "");
             }).get();
 
             final List<DnsRecord> records = result.join();

